CONTRIBUTING.rst

```diff
@@ -83,9 +83,52 @@ Using a custom ``virtualenv``
     $ cd ${HOME}/hack-on-google-cloud-python
     $ ${VENV}/bin/python setup.py install
 
+- To keep your ``virtualenv`` from getting checked-in, you can make a
+  global gitignore file::
+
+  $ echo "${ENV_NAME}" >> ~/gitignore_global
+  $ git config --global core.excludesfile ~/.gitignore_global
+
   Unfortunately using ``setup.py develop`` is not possible with this
   project, because it uses `namespace packages`_.
 
+Using ``virtualenvwrapper``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- To install ``google-cloud-python`` with ``virtualenvwrapper`` you
+  can run the following::
+
+  $ pip install virtualenvwrapper
+  $ export WORKON_HOME=~/Envs
+  $ source /usr/local/bin/virtualenvwrapper.sh
```

Comment:
http://docs.python-guide.org/en/latest/dev/virtualenvs/#virtualenvwrapper

>$ pip install virtualenvwrapper
$ export WORKON_HOME=~/Envs
$ source /usr/local/bin/virtualenvwrapper.sh

That's how they install all the commands. You're supposed to add it to your *rc though.